contentful/content_type_cache.py

```diff
@@ -16,15 +16,15 @@ class ContentTypeCache(object):
     Used for properly coercing Entry fields.
     """
 
-    __CACHE__ = []
+    __CACHE__ = {}
 
     @classmethod
-    def get(cls, content_type_id):
+    def get(cls, space_id, content_type_id):
```

Comment:
Lookin' good!

I think you might need to pass the space_id in 

https://github.dev/contentful/contentful.py/blob/7105eee74d719ea212370ac62479a1c7a660c87d/contentful/entry.py#L52

somehow :thinking:

Reply:
@EricNelson12 thank you for pointing that out.